### PR TITLE
VILO excelfile

### PR DESCRIPTION
**Pull request by Bella2347**
An excelfile is generated containing volumes for how to dilute a sample.
Removed some columns
Changed from 2 decimals to 1

Needs to test and update in ReQtest

### CODE REVIEW

**Comment by Bella2347** [comment]
Small change, no major testing needed.

**Comment by Bella2347** [comment]
Tested here https://claritylims.lund.skane.se/clarity/work-details/317006

File: common/cDNAsynthesis_dilution.py [hunk 1]
```diff
@@ -139,20 +139,21 @@ def main():
 						"Expected concentration (ng/ul)" : expectConc,
 						"RNA (ul)" 			 : rna,
 						"H2O (ul)" 			 : h2o,
-						"Analysis"                       : analysis}
+						"Analysis"                       : analysis,
+						"VILO (ul)"			 : 4}
```
Comment (ErikBergman) [inline common/cDNAsynthesis_dilution.py:143]: This line could use a comment on what the abbreviation VILO means and why it's set to 4

Comment (ErikBergman) [reply common/cDNAsynthesis_dilution.py:143]: In general, perhaps it's a good idea to introduce variables with clear names for all constants and put them at the top of the script:

P190_LOW_CONC_THRESHOLD = 6.25
P190_MID_CONC_THRESHOLD = 100.0
P190_HIGH_CONC_THRESHOLD = 1000.0
P210_LOW_CONC_THRESHOLD = 62.5
P210_HIGH_CONC_THRESHOLD = 125.0
DEFAULT_VILO = 4

File: common/cDNAsynthesis_dilution.py [hunk 2]
```diff
@@ -139,20 +139,21 @@ def main():
...
 
 	f_outfile = xlsxwriter.Workbook(args[ "outfile" ] + '.xlsx')
 	f_outsheet = f_outfile.add_worksheet()
 
-	cell_format_2dec = f_outfile.add_format()
-	cell_format_2dec.set_num_format('0.00')
-	cell_format_2dec.set_border()
-	cell_format_2dec.set_border_color('#D9D9D9')
+	cell_format_1dec = f_outfile.add_format()
```
Comment (ErikBergman) [inline common/cDNAsynthesis_dilution.py:148]: Without the context of the previous commit, the meaning of this variable's name is a bit lost

File: common/cDNAsynthesis_dilution.py (hunk 1 above)
Comment (Bella2347) [reply common/cDNAsynthesis_dilution.py:143]: I don't know what VILO stands for and I think 4 ul is the volume they take from the dilution to the next step. 

Good suggestion with the variable names.

**Comment by Bella2347** [comment]
Latest changes have been tested and documentet on ReQtest
https://secure.reqtest.com/app/tests/test-run/view/view-test-run-details/699088

File: common/cDNAsynthesis_dilution.py [hunk 3]
```diff
@@ -162,22 +170,24 @@ def main():
 	cell_format_blue.set_border()
 	cell_format_blue.set_border_color('#D9D9D9')
 
-	columns = (25, 15, 15, 10, 25, 18, 18, 30, 10, 10, 40)
+#	columns = (25, 15, 15, 10, 25, 18, 18, 30, 10, 10, 40)
+	columns = (25, 20, 16, 16, 10, 10, 10)
```
Comment (ryanjameskennedy) [inline common/cDNAsynthesis_dilution.py:174]: Is this meant to be an index for column numbers?

**Review by ryanjameskennedy — APPROVED** [review]
Checked the ReQtest and had a question. Regardless, LGTM!

**Pull request merged**